devito/operator/operator.py

```diff
@@ -639,6 +639,21 @@ def _soname(self):
         """A unique name for the shared object resulting from JIT compilation."""
         return Signer._digest(self, configuration)
 
+    @cached_property
+    def ccode(self):
+        """
```

Comment:
docstrings (`__doc__` attribute) are inherited as well, so you should drop it here. Can do in the subsequent PR if this one is ready

Reply:
Done in next PR